### PR TITLE
RFC: Improve Rx Observable code generation

### PR DESCRIPTION
**Pull request by RLittlesII**
We want to improve the experience of generating observables from events.  This is a PR to document the request for @glennawatson to build Pharmacist. Use the emoji buttons to show approval, comment otherwise.

### CODE REVIEW

File: reactiveui/0004-reactiveui-events-generator-opencollection.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+We want to improve the experience of generating observables from events.
+
+* We will be hiring @glennawatson to provide professional services via our OpenCollective fund to provide an Rx Event to Observables code generation tool.
+
+* The Observable Generator will scan a given nuget package for any event handlers and convert them to an observable stream
+    * Provide an MSBuild Task 
+    * Provide a global dotnet tool
+    * Pull Pharamcist directly into a csproj
```
Comment (worldbeater) [inline reactiveui/0004-reactiveui-events-generator-opencollection.md]:
```suggestion
    * Pull Pharmacist directly into a csproj
```

**Review by worldbeater — DISMISSED** [review]

**Comment by glennawatson** [comment]
Should be noted if the expense gets approved it will be invoiced to my company Beyond Recognition Pty Ltd.

**Comment by glennawatson** [comment]
Expense submitted. https://opencollective.com/reactiveui/expenses/9185

**Comment by glennawatson** [comment]
Work was delivered to https://GitHub.com/ReactiveUI/pharmacist

**Review by worldbeater — APPROVED** [review]

**Comment by glennawatson** [comment]
Also should be this funding was basically the idea of @michaelstonis who wanted to get me funding for making pharmacist, so the work was already done beforehand.

**Comment by ghuntley** [comment]
Expense approved.

**Pull request merged**